modules/virt-about-top-consumers-projects-vms-nodes.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assembly:
+//
+// * virt/logging_events_monitoring/virt-reviewing-virtualization-overview.adoc
+
+[id="virt-about-reviewing-top-consumers-projects-vms-nodes_{context}"]
+= About top consumers for projects, virtual machines, and nodes
+
+The *Top Consumers* cards display projects, virtual machines or nodes with maximum consumption of a resource. You can select a project, a virtual machine, or a node and view the top five consumers of a specific resource.
+
+The following table shows resources monitored for top consumers.
+
+[cols="1,1"]
+|===
+|*Resources monitored for top consumption* | *Description*
+|CPU
+|Projects, virtual machines, or nodes consuming the most CPU.
+|Memory
+|Projects, virtual machines, or nodes consuming the most memory (in bytes).
+|Used filesystem
+|Projects, virtual machines, or nodes with the highest consumption of filesystems (in bytes).
+|Memory swap
+|Projects, virtual machines, or nodes consuming the most memory pressure when memory is swapped .
+|vCPU wait
+|Projects, virtual machines, or nodes experiencing the maximum wait time (in seconds) for the vCPUs.
+|Storage throughput
+|Projects, virtual machines, or nodes with the highest data transfer rate to and from the storage media (in mbps).
```

Comment:
This value adjusts automatically as well. It could be MBps or GBps depending on the value.

Reply:
I've added a brief explanation here, based on your input.